rt_dashboard/templates/rt_dashboard/history.html

```diff
@@ -23,8 +23,8 @@
 <script type="text/javascript">
 window.addEventListener('load', function() {
     var container = document.getElementById('visualization');
-    var items = new vis.DataSet([ {{rows}} ]);
-    var groups = [ {{groups}} ]
+    var items = new vis.DataSet(JSON.parse(JSON.stringify({{rows}})));
+    var groups = JSON.parse(JSON.stringify({{groups}}));
```

Comment:
This shouldn't be needed - `JSON.parse(JSON.stringify(...))` just makes a clone of the data, but in this case it's not used anywhere else.

Reply:
Template doesn't have access to the JSON string, but to the python data structures. So I'd argue we need this

Reply:
The python data is inside the `{{ }}` and once processed it is turned into json and sent to browser. On the browser side it is first parsed as js then stringified to json string and back again to js. I don't see the benefit of the browser side processing.

Reply:
Yeah, you are right, i fixed this